utils/build/docker/php/parametric/server.php

```diff
@@ -398,11 +401,11 @@ function remappedSpanKind($spanKind) {
     return jsonResponse([]);
 }));
 $router->addRoute('POST', '/trace/otel/flush', new ClosureRequestHandler(function (Request $req) {
-    \DDTrace\flush();
-    dd_trace_internal_fn("synchronous_flush");
-     return jsonResponse([
-         'success' => true
-     ]);
+    $timeout = (arg($req, 'seconds') || 0.1) * 1000; # convert timeout to ms
```

Comment:
```suggestion
    $timeout = (arg($req, 'seconds') ?: 0.1) * 1000; # convert timeout to ms
```
It's not javascript :-)

Reply:
Thanks 😅